BREAKING-CHANGES.md

```diff
@@ -65,6 +65,26 @@ This is a quick summary of the sections below:
   - MSRV is now 1.63.0
   - `List` no longer ignores empty strings
 
+## v0.29.0 (Unreleased)
+
+### `Line` now implements `From<Cow<str>` ([#1373])
+
+[#1373]: https://github.com/ratatui/ratatui/pull/1373
+
+As this adds an extra conversion, ambiguous inferred values may no longer compile.
```

Comment:
Should this be written like this:

```suggestion
As this adds an extra conversion, ambiguously inferred values may no longer compile.
```

?

Reply:
I meant it as an adjective attaching to the noun phrase "inferred values", rather than an an adverb attaching to "inferred". "Values" should be "expressions" here. The concrete part of this is that there is an inferred expression, which is ambiguous when evaluated. I lean 80/20 towards the original wording (but would change it to expression).

Reply:
bd5fa963f..eaa180cc6